CONTRIBUTING.md

```diff
@@ -262,8 +262,8 @@ docker run --rm -v $(pwd):$(pwd):rw -w $(pwd) -v /var/run/docker.sock:/var/run/d
   * **description** -> Description of Appwrite SDK
   * **namespace** -> SDK Namespace
   * **version** -> SDK Version
-  * **endpoint** -> Default Endpoint (example: "https://cloud.appwrite.io/v1")
-  * **host** -> Default Host (example: "cloud.appwrite.io")
+  * **endpoint** -> Default Endpoint (example: "https://appwrite.io/v1")
```

Comment:
Why did this change?

Reply:
Must have been initially fixed in the reverted PR, fixed here